src/components/calcite-accordion/calcite-accordion.tsx

```diff
@@ -0,0 +1,212 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  h,
+  Host,
+  Listen,
+  Prop
+} from "@stencil/core";
+import { UP, DOWN, HOME, END } from "../../utils/keys";
+import { getElementDir } from "../../utils/dom";
+import { guid } from "../../utils/guid";
+
+@Component({
+  tag: "calcite-accordion",
+  styleUrl: "calcite-accordion.scss",
+  shadow: true
+})
+export class CalciteAccordion {
+  //--------------------------------------------------------------------------
+  //
+  //  Element
+  //
+  //--------------------------------------------------------------------------
+
+  @Element() el: HTMLElement;
+
+  //--------------------------------------------------------------------------
+  //
+  //  Public Properties
+  //
+  //--------------------------------------------------------------------------
+
+  /** specify the theme of dropdrown, defaults to light */
+  @Prop({ mutable: true, reflect: true }) theme: "light" | "dark" = "light";
+
+  /** specify the scale of dropdrown, defaults to m */
+  @Prop({ mutable: true, reflect: true }) scale: "s" | "m" | "l" = "m";
+
+  /** specify the appearance, defaults to default */
```

Comment:
Would be helpful to describe what "minimal" does. I can guess but it's not super clear.